core/odp/odps/addNationalClassPlaceHolder.ts

```diff
@@ -1,6 +1,7 @@
-import { ODP, ODPNationalClassFactory } from '@core/odp'
+import { ODPNationalClassFactory } from '@core/odp/odpNationalClass'
+import { OriginalDataPoint } from '@meta/assessment'
```

Comment:
export ODPNationalClassFactory from '@meta/assessment'

Reply:
core is deprecated !!!

Reply:
ok, what about this? import { Objects } from '@core/utils'

Reply:
this is core, so I should leave things as they were

Reply:
we said in the meeting, utils will be moved out ;)